stream_alert/alert_processor/outputs.py

```diff
@@ -291,21 +296,23 @@ def _check_exists_get_id(self, filter_str, url, target_key):
             return False
 
         # If there are results, get the first occurence from the list
-        return response[target_key][0]['id'] if target_key in response else False
+        if get_id:
+            return response[target_key][0]['id'] if target_key in response else False
+
+        return True
 
-    def _user_verify(self, user):
+    def _user_verify(self, user, get_id=True):
```

Comment:
It looks like the only time this function is called, the default value for `get_id` is overridden with `False`... am I missing something? When would `get_id` ever be `True` here?

Reply:
In `_incident_assignment` it gets called and the default `True` value is used